circle.yml

```diff
@@ -18,7 +18,7 @@ test:
     - go tool vet *.go                                                   # Go package in root directory.
     - for d in */; do echo $d; done | grep -v tests/ | grep -v third_party/ | xargs go tool vet # All subdirectories except "tests", "third_party".
     - >
-      gopherjs test --short --minify
+      gopherjs test --short
```

Comment:
As discussed, I'll run a short test for minification to cover it. Say, the `fmt` package:

```yml
- gopherjs test -v --minify fmt # Minification should work.
```

Done.